src/upstream_patches_ui.sh

```diff
@@ -115,8 +113,73 @@ function dashboard_entry_menu()
 # developer. For this reason, we just need to display the save patches.
 function show_bookmarked_patches()
 {
-  # TODO: Get bookmarked_patches from the local database
-  list_patches 'Bookmarked patches' bookmarked_patches
+  if [[ -z "${screen_sequence['RETURNING']}" ]]; then
+    get_bookmarked_series bookmarked_series
+  fi
+
+  # Avoiding stale value
+  screen_sequence['RETURNING']=''
+
+  list_patches 'Bookmarked patches' bookmarked_series "${screen_sequence['SHOW_SCREEN']}"
+}
+
+# Screen resposible for showing a specific bookmarked series details
+#
+# @series_index: Series index
+function show_bookmarked_series_details()
+{
+  local series_index="$1"
+  local -a action_list
+  local patch_metadata
+  local target_patch
+  local message_box
+  local columns
+  local patch_title
+  local download_dir_path
+
+  action_list=('Unbookmark')
+
+  # The local bookmark database starting index is 1 and the index
+  # passed as argument starts at 0.
+  ((series_index++))
```

Comment:
Just to be safe and based on your comment, I recommend you to check `series_index` before trying to increment it.